dev/sparktestsupport/shellutils.py

```diff
@@ -48,7 +48,10 @@ def run_cmd(cmd):
     if not isinstance(cmd, list):
         cmd = cmd.split()
     try:
-        subprocess.check_call(cmd)
+        if return_output:
+            return subprocess.check_output(cmd)
```

Comment:
This is new in 2.7, so we may need to make sure that this script is run by python2.7?